fvtest/porttest/CMakeLists.txt

```diff
@@ -19,35 +19,7 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ###############################################################################
 
-include_directories( ${CMAKE_CURRENT_SOURCE_DIR} )
-
-set(cudaSources "")
-set(socketSources "")
-
-if(OMR_OPT_CUDA)
-	set(cudaSources
-		cudaBasic.cpp
-		cudaError.cpp
-		cudaEvent.cpp
-		cudaInvalid.cpp
-		cudaLaunch.cpp
-		cudaLink.cpp
-		cudaMemory.cpp
-		cudaModule.cpp
-		cudaPeer.cpp
-		cudaProperty.cpp
-		cudaPtx.cpp
-		cudaStream.cpp
-		cudaTests.cpp
-	)
-endif()
-
-# TODO: Remove AND (NOT OMR_HOST_OS STREQUAL "win") after OMRSOCK API is implemented on Windows.
-if(NOT OMR_HOST_OS STREQUAL "win")
-	set(socketSources
-		omrsockTest.cpp
-	)
-endif()
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Rather than use `include_directories` to affect all libs in a directory, I prefer that we set this on a per-target basis, so for example:

```cmake
target_include_directories(omrporttest
	PRIVATE
		${CMAKE_CURRENT_SOURCE_DIR}
)
```

Reply:
this would have to come _after_ `omrporttest` is defined.

Reply:
Sounds good, I will make this change.

Reply:
Actually it gave an error of:
```
  Cannot specify include directories for target "omrporttest" which is not
  built by this project.
```

Reply:
Actually, I think moving it after the first omr_add_executable solves this problem. I will try that.

Reply:
Oh sorry, you mentioned that :+1: Yes it works.